test/functional/wallet_import_with_label.py

```diff
@@ -119,7 +119,7 @@ def run_test(self):
         )
         priv_key4 = self.nodes[0].dumpprivkey(address4)
         self.nodes[1].importprivkey(priv_key4)
-        embedded_addr = self.nodes[1].getaddressinfo(address4)['embedded']['address']
+        embedded_addr = self.nodes[1].getaddressinfo(address4)["embedded"]['address']
```

Comment:
in commit 8e82acc8ade: Why?

Reply:
I had removed it, then manually reverted with my favorite quote: the double quote. fixed

Reply:
> my favorite quote: the double quote

there are `10` types of people in this world..